tests/ansibletest.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -eu
+
+mkdir -p ansible_collections/cyberark/
+cd ansible_collections/cyberark/
+git clone --single-branch --branch deleteit_1 https://github.com/cyberark/ansible-conjur-collection.git
```

Comment:
Why do we need to clone the repo here? If this script is run from Jenkins, the repo should be available to the script already.

Reply:
As per my understanding , the Ansible-test working directory should be like "/ansible_collections/{namespace}/{collection}/" but we dont have it . That's why I put our repo in this directory structure .